agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -457,6 +458,31 @@ function install_k8s_agent {
 \1  sysdig-instance: '$SYSDIG_INSTANCE_NAME'/' $DAEMONSET_FILE
     fi
 
+    # add the bpf configs if enabled
+    if [[ ($BPF -eq 1) ]]; then
+		echo "* Enabling eBPF"
+		sed -i.bak -e "s|#env:|env:|" $DAEMONSET_FILE
```

Comment:
This will only work if there's one commented-out `env` section in the daemonset.

Reply:
thanks! I copied this section from: https://github.com/draios/agent/blob/dev/scripts/jenkins/repo-upload/install-agent-kubernetes
where are their daemonset configs hosted? I'd like to check them.

Reply:
I don't know. I'm not an IBM guy or a k8s guy.

My first thought is that maybe we could match the entire block using one sed script to avoid this problem. Another possibility would be to use a comment after `env:` to help match the correct one (something like `#env: #BPF`)

Reply:
Since this uses the same yamls as the regular install-agent-kubernetes script (from agent_deploy/kubernetes in this repo), anything copied over from there should at most only be as broken as the original 😄 

I would guess that there were some time or tool limitations that led to this approach at the time. I like the idea Nathan suggested of matching the whole commented block, and it'll be more clear since rather than uncommenting all occurrences of a specific line, like `#  - name: SYSDIG_BPF_PROBE`, you'll have to specify each commented block separately.

The best approach IMO would be to ignore the commented blocks altogether, just using some sort of anchor/tag to find the specific sections to insert yaml blocks where needed. That may require too many changes for this story though.

Reply:
thanks! shall I merge?